discord/ui/view.py

```diff
@@ -178,7 +178,7 @@ def _init_children(self) -> List[Item[Self]]:
             children.append(item)
         return children
 
-    def __init__(self, *, timeout: Optional[float] = 180.0):
+    def __init__(self, *items: Item, timeout: Optional[float] = 180.0):
```

Comment:
```suggestion
    def __init__(self, *items: Optional[Item], timeout: Optional[float] = 180.0):
```
Also edit the docs at line 150:
```
    *items: Optional[:class:`Item`]
        The items to add to the view.

```

Reply:
This doesn't look like the right use of `Optional`.
It should however be annotated as `Item[Any]`

Reply:
Updated